src/doc/rustc/src/codegen-options/index.md

```diff
@@ -22,6 +22,13 @@ This flag lets you append a single extra argument to the linker invocation.
 This flag lets you append multiple extra arguments to the linker invocation. The
 options should be separated by spaces.
 
+## linker-flavor
+
+This flag lets you control the linker flavor used by `rustc`. If a linker is given `-C linker`
```

Comment:
```suggestion
This flag lets you control the linker flavor used by `rustc`. If a linker is given the `-C linker`
```

Reply:
I've updated the PR with improved wording. It isn't exactly your suggestion but it is a similar improvement. Thanks.